sdk/translation/ai-translation-text-rest/src/textTranslationClient.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
This file shall be replaced by `customClient` - see its implementation, it contains createClient